### PR TITLE
PS-8889 Add quick start topic (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/quickstart-docker.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/quickstart-docker.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+    ```{.text .no-copy}
+    9d0acf24fcf66ad01fcc19067206eada4e2303dad039bb020daa2bd0b2e55d59
+    ```
+
+We have added the following options:
+
+* `--name` - provides a selected name to the container. If you do not use this option, docker provides a random name.
+* `-d` - the container runs in the background in detached mode.
+* `-e` - to start a Percona Server for MySQL container use the `MYSQL_ROOT_PASSWORD` environment variable. The instance refuses to initialize without these variables. Choose a more secure password, if needed.
```
Comment (alina-derkach-oaza) [inline docs/quickstart-docker.md]:
```suggestion
* `-e` - to start a Percona Server for MySQL container, use the `MYSQL_ROOT_PASSWORD` environment variable. The instance refuses to initialize without this variable. Choose a more secure password if needed.
```

File: docs/quickstart-docker.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+Connect with the `docker exec` command to run commands in a running container.
+
+```{.bash data-prompt="$"}
+$ docker exec -it psmysql /bin/bash
+```
+
+We have the following options:
+
+* `it` - Interactive and be a pseudo-terminal
```
Comment (alina-derkach-oaza) [inline docs/quickstart-docker.md]:
```suggestion
* `it` - interactive and be a pseudo-terminal
```

File: docs/quickstart-docker.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+
+    ```{.text .no-copy}
+    CONTAINER ID   IMAGE                           COMMAND                  CREATED              STATUS              PORTS                 NAMES
+    9d0acf24fcf6   percona/percona-server:8.0.34   "/docker-entrypoint.…"   About a minute ago   Up About a minute   3306/tcp, 33060/tcp   psmysql
+    ```
+
+You can inspect the volume.
+
+```{.bash data-prompt="$"}
+docker inspect psmysql | grep -i volume
```
Comment (alina-derkach-oaza) [inline docs/quickstart-docker.md]:
```suggestion
$ docker inspect psmysql | grep -i volume
```

File: docs/quickstart-docker.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+
+
+## Connect to the instance
+
+Connect to the database server instance with the following command:
+
+```{.bash data-prompt="$"}
+$ mysql -uroot -p
+```
+
```
Comment (alina-derkach-oaza) [inline docs/quickstart-docker.md:33]: We need to add explanations about the password. that we use secret

Comment (patrickbirch) [reply docs/quickstart-docker.md:33]: Line 155
* `-p` - the password to use when connecting to the server. Enter the password when prompted by the server.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**